libs/components/src/accordion/stories/vl-accordion.stories.ts

```diff
@@ -79,3 +81,12 @@ AccordionTitleSlot.args = {
         '<span>Onderwijs helpt jonge mensen en volwassenen om zichzelf te ontwikkelen en hun weg te vinden in onze samenleving. Het hoger onderwijs speelt daarnaast een belangrijke rol in innovatie dankzij het belang van wetenschappelijk onderzoek.</span>',
     titleSlot: '<span slot="title">Lees meer over de onderwijsdoelstelling</span>',
 };
+
+export const AccordionDefaultOpen = Template.bind({});
```

Comment:
we gaan typisch niet voor elk attribuut een nieuwe story maken; wat denk jij @ksvlaanderen 

ben niet zeker of dat hier nuttig is

Reply:
Heb me die vraag ook gesteld. Maar net omdat het een initiële "state" anders maakt heb ik aparte story gemaakt

Reply:
hmm you sure.. "binnenkort"? Gaan jullie over alles heen gaan om dat na te gaan? (is ok voor mij hé)

Reply:
Ok; besproken met Kristof en akkoord om niet voor alles afzonderlijke test te voorzien (behalve bij side-effects misschien)